cmd/consensus/main.go

```diff
@@ -460,13 +460,18 @@ func main() {
 			}
 			committee = committees.NewMetricsWrapper(committee, mainMetrics) // wrapper for measuring time spent determining consensus committee relations
 
+			epochLookup := epochs.NewEpochLookup(node.State)
+
+			signerStore := signature.NewEpochAwareSignerStore(epochLookup, node.Storage.DKGKeys)
+
 			// initialize the combined signer for hotstuff
 			var signer hotstuff.SignerVerifier
 			signer = verification.NewCombinedSigner(
 				committee,
 				staking,
-				beacon,
+				verifier,
```

Comment:
The name `verifier` also applies to the staking module, I think it's better to specify `beacon` or `threshold` here too.